src/cls/tabular/cls_tabular.cc

```diff
@@ -1908,17 +1907,8 @@ int transform_db_op(cls_method_context_t hctx, bufferlist *in, bufferlist *out)
     }
 
     CLS_LOG(20, "transform_db_op: table_name=%s", op.table_name.c_str());
-    CLS_LOG(20, "transform_db_op: data_schema=%s", op.data_schema.c_str());
     CLS_LOG(20, "transform_db_op: transform_format_type=%d", op.required_type);
 
-    // Check if transformation is required or not
-    if (format_type == op.required_type) {
-        // Source and destination object types are same, therefore no tranformation
-        // is required.
-        CLS_LOG(20, "No Transforming required");
-        return 0;
-    }
-
     // Object is sequence of actual data along with encoded metadata
     bufferlist encoded_meta_bls;
     int ret = cls_cxx_read(hctx, 0, 0, &encoded_meta_bls);
```

Comment:
add TODO: get individ off/len of fbmeta's inside obj and read one at a time.

Reply:
Done